pkgs/applications/emulators/fs-uae/launcher.nix

```diff
@@ -15,29 +10,26 @@ stdenv.mkDerivation rec {
     sha256 = "1dknra4ngz7bpppwqghmza1q68pn1yaw54p9ba0f42zwp427ly97";
   };
 
-  nativeBuildInputs = [
-    gettext
-    makeWrapper
-    python3
-  ];
+  nativeBuildInputs = [ gettext python3 wrapQtAppsHook wrapGAppsHook ];
```

Comment:
Are both gtk and qt really required? The issue you linked only mentions qt. I've never seen a program use both of these frameworks before.

Reply:
Nope, you are right. The gtk wrapper for sway on my system broke, so I'm seeing an error.